src/codegen/object.rs

```diff
@@ -440,7 +440,6 @@ impl {name}Builder {{
         "
     // rustdoc-stripper-ignore-next
     /// Build the [`{name}`].
-    #[must_use = \"The builder must be built to be used\"]
     pub fn build(self) -> {name} {{
         let mut properties: Vec<(&str, &dyn ToValue)> = vec![];",
```

Comment:
But it could also make sense to mark the builder type as `#[must_use]`, see the [docs](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute):

> When used on user-defined composite types, if the expression of an expression statement has that type, then the unused_must_use lint is violated.

I.e. if you forget to call `build()` then you have an expression statement that has the type of the builder.

Reply:
See https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d1080d58a2abe179ca6483ef685bdd64